### ISSUE TITLE
MAF munging

### ISSUE THREAD

**Issue by bschilder** [omerwe/polyfun#10]
# munge_polyfun_sumstats.py

In line 62 I noticed that when the columns are renamed, freq and MAF seem to be treated the same. But couldn't these two things be different in a summary stats file? Perhaps one way would be to check if 1-freq ≤ .5, and if it is then you know it's the minor allele (and then can flip the ref/alt alleles and effect, though the specifics of this might depend on the particular file format).

**Comment by omerwe** [comment]
Good point! The code actually treats MAF as FREQ (does not assumes it's <0.5), so the name is wrong but the functionality is correct. I'll keep this issue open to remind myself to change the MAF name...

**Comment by bschilder** [comment]
Got it, thanks for clarifying!

**Comment by bschilder** [comment]
Great! Happy to put the shell script together, will share it as soon as it's ready.
